libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -349,6 +335,25 @@ bool AsmAnalyzer::operator()(Block const& _block)
 	return success;
 }
 
+bool AsmAnalyzer::expectExpression(Statement const& _statement)
+{
+	bool success = true;
+	int const initialHeight = m_stackHeight;
+	if (!boost::apply_visitor(*this, _statement))
+		success = false;
+	if (m_stackHeight - initialHeight != 1)
+	{
+		m_errorReporter.typeError(
+			locationOf(_statement),
+			"Expected instruction(s) to deposit one item to the stack but did deposit " +
+			boost::lexical_cast<string>(m_stackHeight - initialHeight) +
+			" items."
```

Comment:
haha, I thought about this :-)
The check is always against one and this about a failure, so it is either `0 items` or `2 items` or more :-)